app/components/comparison-area-selector.js

```diff
@@ -28,7 +58,7 @@ export default Ember.Component.extend({
   nestedOptions(optsList) {
     return optsList
       .then(options => nest()
-        .key(d => d.geotype)
+        .key(d => d.typelabel)
```

Comment:
This may conflict with @allthesignals's `350-row-models` branch.

Reply:
Actually, maybe not. This isn't a table row. I was just searching for `d => d.foo` everywhere.